Gemfile

```diff
@@ -16,7 +16,14 @@ gem 'webpacker', '~> 4.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+gem 'hirb', '~> 0.7.3'
 gem 'jbuilder', '~> 2.7'
+
+gem 'bcrypt-ruby', '~> 3.1', '>= 3.1.5'
```

Comment:
## Name change
- [ ] The name for the gem has changed. Would you mind changing it to the suggestion below for efficiency in the dependencies installation process, please?
```suggestion
gem 'bcrypt', '~> 3.1', '>= 3.1.16'
```